theme/templates/accounts/profile.html

```diff
@@ -358,6 +358,73 @@ <h5><span class="text-muted">Date: </span>{{ res.created }}</h5>
+                                                            <div class="progress-bar progress-bar-danger" 
+                                                            style="width: {%if qd.percent_over > 100%}100{%else%}{{qd.percent_over}}{%endif%}%"
+                                                                title="{{ qd.percent_over|floatformat:"1" }}% over allocated space">
+                                                                <span class="sr-only">{{ qd.percent_over|floatformat:"1" }}% over allocated space</span>
+                                                            </div>
+                                                        </div>
+                                                        <span class="text-danger">You are over your allocated quota by {{ qd.percent_over|floatformat:"1" }}%</span>
+                                                    {% else %}
+                                                        <div class="progress">
+                                                            <div class="progress-bar progress-bar-success" 
+                                                                style="width: {{ qd.uz_percent }}%" 
+                                                                title="{{ qd.uz_percent|floatformat:"1" }}% used by userZone">
+                                                                <span class="sr-only">{{ qd.uz_percent|floatformat:"1" }}% of quota used by userZone</span>
+                                                            </div>
+                                                            <div class="progress-bar progress-bar-info" 
+                                                                style="width: {{ qd.dz_percent }}%"
+                                                                title="{{ qd.dz_percent|floatformat:"1" }}% used by dataZone">
+                                                                <span class="sr-only">{{ qd.dz_percent|floatformat:"1" }}% of quota is used by dataZone</span>
+                                                            </div>
+                                                        </div>
+                                                        <span>{{ qd.remaining }}{{ qd.unit }} free of {{ qd.allocated }}{{ qd.unit }}</span>
+                                                    {% endif %}
+                                                </div>
+                                                {% endwith %}
+                                                {% if quota_message %}
+                                                    <div class="alert alert-info">
+                                                        {{ quota_message }}
+                                                        Additional Quota information can be found <a href="https://help.hydroshare.org/about-hydroshare/policies/quota/" target="_blank">here.</a>
```

Comment:
I think this link needs to be changed to 'Read more about Additional Quota', where 'Additional Quota' is a link and has a bold font.

See https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#link_best_practices

Reply:
good call thanks